Resources/ServerInfo/Rules.txt

```diff
@@ -2,12 +2,17 @@
 
 [color=#99ff99]Frontier Station is a Medium Roleplay server. Try to immerse yourself into your character. This includes doing your job as a station or ship crew member and delegating tasks that are outside of that description, or striking out upon space as a Captain of your ship. This does not imply you’re forced to play the whole round; Frontier Station is designed with pop-in-and-out gameplay in mind, where some players will come and go but their bank account remains persistent across server restarts.[/color]
 
+[color=#99ff99]You can access more information about the game using the NUMPAD 0 Guidebook.[/color]
```

Comment:
I think this reads better if NUMPAD 0 is in parentheses after "Guidebook".

Ideally I think this should have a reference to the buttons at the top of the screen, but I can't figure out how to write that succinctly ("Question mark button in the top menu?").
```suggestion
[color=#99ff99]You can access more information about the game using the Guidebook (NUMPAD 0).[/color]
```

Reply:
Ideally the rules pages should  link to the guidebook, im sure next upstream merge will have the whole rules page moved to the guidebooks too but this will do for now.